### DISCUSSION TITLE
TrustedCluster Compatibility Version

### DISCUSSION THREAD

**Discussion by mukitaro** [gravitational/teleport#11723] [Q&A] [answered]
Hello,

I have a question about TrustedCluster support version.

If we build a leaf cluster with teleport8 to access CentOS6 nodes.
Does the root cluster version only support the next major version(teleport9)?

Or does TrustedCluster support the difference two or more major versions(=> teleport10)?

Thanks!

**Comment by webvictim** [comment, marked as answer]
Trusted clusters should have the same version guarantees as Teleport i.e. n-1 version. Teleport 8 leaf connecting to Teleport 9 root would theoretically be supported, but a Teleport 8 leaf connecting to Teleport 10 root would not be.

**Reply by mukitaro** [reply]
Hi webvictim

Thank you for providing an answer to my question.
I understood it.
